app/scripts/migrations/052.js

```diff
@@ -0,0 +1,21 @@
+import { cloneDeep } from 'lodash';
+
+import { removeStorageItem } from '../../../ui/lib/storage-helpers';
+
+const version = 52;
+
+/**
+ * Removes storage key no longer required due to move to
+ * fetchWithCache for gas retrieval and storage
+ */
+export default {
+  version,
+  async migrate(originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData);
+    versionedData.meta.version = version;
+
+    await removeStorageItem('BASIC_PRICE_ESTIMATES_LAST_RETRIEVED');
```

Comment:
Should the `BASIC_PRICE_ESTIMATES` key be removed as well? It also appears to now be unused.

Reply:
Good shout!